cfgov/cfgov/settings/base.py

```diff
@@ -398,28 +402,52 @@
 
 PRIVACY_EMAIL_TARGET = os.environ.get("PRIVACY_EMAIL_TARGET", "test@localhost")
 
-
 # Password Policies
-# cfpb_common password rules
-CFPB_COMMON_PASSWORD_RULES = [
-    [r".{12,}", "Minimum allowed length is 12 characters"],
-    [r"[A-Z]", "Password must include at least one capital letter"],
-    [r"[a-z]", "Password must include at least one lowercase letter"],
-    [r"[0-9]", "Password must include at least one digit"],
-    [
-        r"[@#$%&!]",
-        "Password must include at least one special character (@#$%&!)",
-    ],
+AUTH_PASSWORD_VALIDATORS = [
+    {
+        "NAME": "django.contrib.auth.password_validation.MinimumLengthValidator",
+        "OPTIONS": {
+            "min_length": 12,
+        },
+    },
+    {
+        "NAME": "login.password_validation.ComplexityValidator",
+        "OPTIONS": {
+            "rules": [
+                [
+                    r"[A-Z]",
+                    "Password must include at least one capital letter",
+                ],
+                [
+                    r"[a-z]",
+                    "Password must include at least one lowercase letter",
+                ],
+                [r"[0-9]", "Password must include at least one digit"],
+                [
+                    r"[@#$%&!]",
+                    "Password must include at least one special character (@#$%&!)",
+                ],
+            ],
+        },
+    },
+    {
+        "NAME": "login.password_validation.HistoryValidator",
+    },
+    {
+        "NAME": "login.password_validation.AgeValidator",
+    },
 ]
-# cfpb_common login rules
-# in seconds
-LOGIN_FAIL_TIME_PERIOD = int(
-    os.environ.get("LOGIN_FAIL_TIME_PERIOD", 120 * 60)
+
+# Login lockout rules using django-axes
+AUTHENTICATION_BACKENDS = (
+    "axes.backends.AxesStandaloneBackend",
+    "django.contrib.auth.backends.ModelBackend",
 )
-# number of failed attempts
-LOGIN_FAILS_ALLOWED = int(os.environ.get("LOGIN_FAILS_ALLOWED", 5))
-LOGIN_REDIRECT_URL = "/admin/"
-LOGIN_URL = "/login/"
+AXES_ENABLED = True
+AXES_FAILURE_LIMIT = 5
```

Comment:
I notice that you didn't bring over `LOGIN_FAILS_ALLOWED` -- since both this and `LOGIN_FAIL_TIME_PERIOD` have been essentially fixed since their creation years ago, I propose hardcoding them here in settings and deprecating them from the deployment config, in the interest of simplicity. Having them be configurable by environment variable doesn't add any benefit since they get defined in settings whether or not the variable is set or not.

Reply:
Done in [4b10d93](https://github.com/cfpb/consumerfinance.gov/pull/7650/commits/4b10d9313097dfc9b9f941f78742bc5e4c275ed2)